### ISSUE TITLE
Black Screen

### ISSUE THREAD

**Issue by Zeon556g** [mmtrt/leagueoflegends#206]
$ MESA-INTEL: warning: Performance support disabled, consider sysctl dev.i915.perf_stream_paranoid=0

MESA-INTEL: warning: Performance support disabled, consider sysctl dev.i915.perf_stream_paranoid=0

fixme:vkd3d_create_device: Unsupported feature level 0xc100.
MESA-INTEL: warning: Performance support disabled, consider sysctl dev.i915.perf_stream_paranoid=0

MESA-INTEL: warning: Performance support disabled, consider sysctl dev.i915.perf_stream_paranoid=0

fixme:vkd3d_create_device: Unsupported feature level 0xc000.
MESA-INTEL: warning: Performance support disabled, consider sysctl dev.i915.perf_stream_paranoid=0

MESA-INTEL: warning: Performance support disabled, consider sysctl dev.i915.perf_stream_paranoid=0

wine: Unhandled exception 0xe0000008 in thread 114 at address 7B035485 (thread 0114), starting debugger...
LeagueClientUx pid: 11068
Waiting for port 33257
02cc:err:winediag:MIDIMAP_drvOpen No software synthesizer midi port found, Midi sound output probably won't work.

norman@norman-TUF-Gaming-FX505GT-FX505GT:~$ MESA-INTEL: warning: Performance support disabled, consider sysctl dev.i915.perf_stream_paranoid=0

MESA-INTEL: warning: Performance support disabled, consider sysctl dev.i915.perf_stream_paranoid=0

fixme:vkd3d_create_device: Unsupported feature level 0xc100.
MESA-INTEL: warning: Performance support disabled, consider sysctl dev.i915.perf_stream_paranoid=0

MESA-INTEL: warning: Performance support disabled, consider sysctl dev.i915.perf_stream_paranoid=0

fixme:vkd3d_create_device: Unsupported feature level 0xc000.
MESA-INTEL: warning: Performance support disabled, consider sysctl dev.i915.perf_stream_paranoid=0

MESA-INTEL: warning: Performance support disabled, consider sysctl dev.i915.perf_stream_paranoid=0

fixme:vkd3d_create_device: Unsupported feature level 0xb100.
MESA-INTEL: warning: Performance support disabled, consider sysctl dev.i915.perf_stream_paranoid=0

MESA-INTEL: warning: Performance support disabled, consider sysctl dev.i915.perf_stream_paranoid=0

fixme:vkd3d_select_physical_device: Multiple physical devices available, selecting the first one.

**Comment by mmtrt** [comment]
Is this client black screen after login OR its after champ selection timeout?

**Comment by alizag** [comment]
I am running into a similar issue, I followed the installation steps (including the optional section) on Pop OS 22.04 LTS.  I am running Nvidia RTX 2080 Super with Nvidia driver version: 4.6.0 NVIDIA 510.68.02.  

The black screen happens right after login after I click 'play' button for LoL on the Riot Client.  When I ran the game from terminal I got the following error messages:
`
01ec:err:rpc:I_RpcGetBuffer no binding
04a0:err:user:load_desktop_driver failed to load L"C:\\windows\\system32\\winex11.drv"
04a0:err:vulkan:wine_vk_init Failed to load Wine graphics driver supporting Vulkan.
`
